docs/tech/api-change-management.md

```diff
@@ -0,0 +1,15 @@
+# API Change management
+
+Rationale of our change management policy is to be backward compatible on existing features
+(classes, methods or [configuration](../configuration.md) fields) whenever possible.
```

Comment:
it might be worth linking to https://developers.symphony.com/restapi/docs/api-change-management to explain what we do in terms of REST API that are called by the BDK